File: testprojects/streaming/python/pipelines/StreamingTestPipeline_1Clone/code/streamingtestpipeline_1/graph/Filter_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from streamingtestpipeline_1.config.ConfigStore import *
from streamingtestpipeline_1.udfs.UDFs import *

def Filter_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.filter(lit(True))

File: testprojects/master_scala_mainone/pipelines/SCALA_DEPENDENCY_MANAGEMENT/code/src/main/scala/graph/everythingSG_1/Filter_1.scala
package graph.everythingSG_1

import io.prophecy.libs._
import udfs.UDFs._
import graph.everythingSG_1.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Filter_1 {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.filter(col("`c  - int`") > lit(-100))

}
